ocp_resources/storage_profile.py

```diff
@@ -4,4 +4,30 @@
 
 
 class StorageProfile(Resource):
+    """
```

Comment:
please add resource info according to https://github.com/RedHatQE/openshift-python-wrapper/blob/main/CONTRIBUTING.md#adding-a-new-module-resource

this is not a new resource but the info should be added to it

Reply:
Can a user create a custom storageprofile without creating storageclass?
I'm not sure if "init" fits here

Reply:
relationship is 1:1 for StorageClass : StorageProfile
so:
- when StorageClass is created, StorageProfile is also added
- adding StorageProfile without esixstingSC, it will be deleted
- adding StorageProfile with esixstingSC, it will be deleted